TESTS/mbedmicro-rtos-mbed/MemoryPool/main.cpp

```diff
@@ -0,0 +1,608 @@
+#include "mbed.h"
```

Comment:
Can you please add the license header here, every file should have it

Reply:
Licence header has been added.